draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -607,15 +607,15 @@ as enhanced by {{RFC1123}}, Section 2.1. Thus, domain-value is a string of
 {{Section 2.3.2.1 of RFC5890}}.
 
 The portions of the set-cookie-string produced by the cookie-av term are
-known as attributes. To maximize compatibility with user agents, servers SHOULD
+known as attributes. To maximize compatibility with user agents, servers MUST
 NOT produce two attributes with the same name in the same set-cookie-string.
```

Comment:
If there is a prohibition on folding multiple Set-Cookie fields into one, does this need to be reformulated?  (set-cookie-string, as I understand it, is the formulation for a single field value instance)

Reply:
This requirement covers attributes (the repeated `cookie-av` grammar) within a single field value instance. That is, servers ought not produce something like `Set-Cookie: name=value;SameSite=Strict;SameSite=Lax`. We could reword this as "... servers MUST NOT produce a `set-cookie-string` with more than one `cookie-av` of a given type", but it's not clear to me that that's any more helpful. 🤷 Do you have a suggestion about how we could word this more explicitly?

Reply:
Ah, that was probably more of a misread on my part.  This seems good once you corrected my misconception.